VTTiny/Components/Animator/IPackageAble.cs

```diff
@@ -0,0 +1,12 @@
+﻿using VTTiny.Components.Animator.Data;
+
+namespace VTTiny.Components.Animator;
+
+public interface IPackageAble
```

Comment:
I think the proper capitalization here should be `IPackageable`, in order to fit with everything else.